README.md

```diff
@@ -44,8 +50,10 @@ Getting started
 
 Running PaStA
 -------------
+### Modes
+**PaStA** has 2 main modes of operation, Mailbox mode and Patch Stack mode.
```

Comment:
has two modes of operation, mailbox (mbox) mode and patch stack mode.

Reply:
Sure